### PR TITLE
docs: add supported language versions to README

### PR DESCRIPTION
**Pull request by childish-sambino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,33 @@
...
+```csharp
+System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
+```
+
+### Supported .NET versions
 
 This library supports .NET applications written in C#, VB.Net, and F# that utilize .NET Framework 3.5 or above or any [supported version](https://dotnet.microsoft.com/platform/support/policy/dotnet-core#lifecycle) of .NET Core (.NET Standard v1.4).
 
+## Installation
+
```
Comment (eshanholtz) [inline README.md]: Remove empty section

Comment (childish-sambino) [reply README.md]: It's not empty if it contains sub-sections.

Comment (eshanholtz) [reply README.md]: Ah. See comment below

File: README.md [hunk 2]
```diff
@@ -4,16 +4,33 @@
...
 ## Adding Twilio libraries to your .NET project
```
Comment (eshanholtz) [inline README.md]: `### Adding Twilio libraries to your .NET project`

Comment (childish-sambino) [reply README.md]: 🤦‍♂

Comment (childish-sambino) [reply README.md]: Updated.

**Pull request merged**